src/lib/key_cache/key_cache.ml

```diff
@@ -109,14 +109,13 @@ module Sync : S with module M := Or_error = struct
               Error.createf "died after receiving %s (signal number %d)"
                 (Signal.to_string s) (Signal.to_system_int s) )
         |> Result.map_error ~f:(fun err ->
-               [%log debug] "Could not download key to key cache"
+               [%log trace] "Could not download key to key cache"
                  ~metadata:
                    [ ("url", `String uri_string)
-                   ; ("local_file_path", `String file_path)
-                   ; ("err", Error_json.error_to_yojson err) ] ;
```

Comment:
Should we keep this in the case the error is anything other then file-not-found?

Reply:
I don't think we need to: everything is deterministic now, so worst case is we fall back to re-generating the keys and then everything proceeds as normal.